cypress/e2e/testtab.cy.js

```diff
@@ -171,11 +173,18 @@ function copyTheReportsToTestTab() {
   cy.wait(100);
   cy.checkFileTreeLength(2);
   cy.wait(100);
-  cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(0).find('.item-name').eq(0).click();
+  // TODO: Uncomment this line when the sequence in the debug tree has been fixed.
+  // cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(0).find('.item-name').eq(0).click();
+  // TODO: And at that time also remove the line below.
+  cy.get('[data-cy-debug-tree="root"] > app-tree-item').contains('Simple report').click()
   cy.wait(100);
   cy.debugTreeGuardedCopyReport('Simple report', 3, 'first');
   cy.wait(100);
-  cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(1).find('.item-name').eq(0).click();
+  if(testSequence) {
+    cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(1).find('.item-name').eq(0).click();
+  } else {
+    cy.get('[data-cy-debug-tree="root"] > app-tree-item').contains('Another simple report').click()
+  }
```

Comment:
I think you forgot to remove this if else block?

Reply:
Gedaan.